packages/minter-contracts/ligo/src/minter_collection/editions/fa2_multi_nft_token_editions.mligo

```diff
@@ -2,6 +2,10 @@
 #include "../nft/fa2_multi_nft_manager.mligo"
 #include "../nft/fa2_multi_nft_asset_simple_admin.mligo"
 
+#if OWNER_HOOKS
+"Sorry but this contract does not work with hooks yet" 
```

Comment:
I would also mention a raw list of things that have to be adjusted for the contract to work with hooks. Something like

```
At the moment of writing, the following places have to be adjusted for hooks to work as expected:
* ...
```

Otherwise, if someone needs this working with hooks eventually, they will have hard time figuring out what needs to be fixed (is it a fundamental limitation? how many places should be fixed? Am I missing something?)

Reply:
I believe we would either have to introduce an inefficient `concat` function to combine the returned list operations for each `edition_collection` we attempt to distribute for OR rework the code so that we can pass the `mint_token_param`s for all editions (across edition collections) to `mint_editions_to_addresses` at once. Either option sacrifices efficiency unnecessarily. I will add this though thanks.